plugins/by-name/blink-cmp/default.nix

```diff
@@ -21,23 +21,7 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
     keymap =
       defaultNullOpts.mkNullableWithRaw (with types; either (attrsOf anything) (enum [ false ]))
         {
-          show = "<C-space>";
-          hide = "<C-e>";
-          accept = "<Tab>";
-          select_prev = [
-            "<Up>"
-            "<C-p>"
-          ];
-          select_next = [
-            "<Down>"
-            "<C-n>"
-          ];
-          show_documentation = "<C-space>";
-          hide_documentation = "<C-space>";
-          scroll_documentation_up = "<C-b>";
-          scroll_documentation_down = "<C-f>";
-          snippet_forward = "<Tab>";
-          snippet_backward = "<S-Tab>";
+          preset = "default";
         }
```

Comment:
Maybe this would benefit from an `example`?

This can be done using `defaultNullOpts.mkNullableWithRaw'`, e.g.

```nix
defaultNullOpts.mkNullableWithRaw' {
  type = with types; either (attrsOf anything) (enum [ false ]);
  pluginDefault = { preset = "default"; };
  example = {
    # Some example! 😁
  };
  description = ''
    # Some description!
  '';
}
```

---

Also: is `enum [false]` still correct?

Reply:
Makes sense, forgot the example part. Will correct this tomorrow, thanks!